### ISSUE TITLE
Latest aframe master targets WebXR - 

### ISSUE THREAD

**Issue by shaneharris** [webmixedreality/exokit#635]
these lines in aframe:

`cannot read 'supportsSession' of undefined`
https://github.com/aframevr/aframe/blob/a35c3676b176d094be5b0a66b6750675910a26e3/src/utils/device.js#L6 

`cannot read property 'isPresenting' of undefined`
https://github.com/aframevr/aframe/blob/62636285a4b7816defda7c26765e1e89b04ec96a/src/core/scene/loadingScreen.js#L19 

I think its around here in [exokit/src/XR.js](https://github.com/webmixedreality/exokit/blob/master/src/XR.js#L33-L35)

I was upgrading to the latest aframe as it solves 3-4 other issues i have - so i hope this is an easy one 🙏

**Comment by shaneharris** [comment]
Slightly older aframe version and it only renders in one eye on oculus through steam.

`e.vr.setPoseTarget is not a function` - this error is also in the chrome browser so might be red herring

Repro 'node . https://maple-bay-1.glitch.me/' 

https://github.com/aframevr/aframe/blob/a35c3676b176d094be5b0a66b6750675910a26e3/src/core/scene/a-scene.js#L588

**Comment by modulesio** [comment]
Thanks! My first guess is this is Three.js WebXR changes trickling downstream.

**Comment by ngokevin** [comment]
Yeah, `e.vr.setPoseTarget is not a function` happens in older A-Frame because three.js detects XR support and tries to use it. In older A-Frame's, I test Exokit to WebVR only.

Latest A-Frame does have XR support if it exists. Might be able to take a look next week.

**Comment by modulesio** [comment]
https://github.com/webmixedreality/exokit/blob/d7f1bbbde68bd08d986e0c2f471da9ea5d76f9c0/src/XR.js#L34
